package.json

```diff
@@ -20,6 +20,7 @@
     "rules": {
       "react/sort-comp": 0,
       "react/no-danger": 0,
+      "react/jsx-no-bind": 0,
```

Comment:
Is anyone against this getting silence? Granted, many of them are my fault, but we have dozens of warnings that I don't think are important to anyone these days?